src/models.py

```diff
@@ -0,0 +1,101 @@
+from datetime import datetime
+
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.dialects.postgresql import UUID
+
+
+db = SQLAlchemy()
+
+class UserLogin(db.Model):
+    __tablename__ = 'user_login'
+    
+    id = db.Column(UUID(as_uuid=True), primary_key=True)
+    password_hash = db.Column(db.String(200))
+    google_login = db.Column(db.Boolean, default=False)
+    last_login = db.Column(db.DateTime, default=datetime.now())
+    is_active = db.Column(db.Boolean, default=True, nullable=False)
+    user_profile_id = db.Column(UUID(as_uuid=True), db.ForeignKey("user_profile.id"))
+    created_at = db.Column(db.DateTime, default=datetime.now(), nullable=False)
+    updated_at = db.Column(db.DateTime, onupdate=datetime.now())
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+
+class UserProfile(db.Model):
+    __tablename__ = 'user_profile'
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True)
+    first_name = db.Column(db.String(254), nullable=False)
+    last_name = db.Column(db.String(254), nullable=False)
```

Comment:
Let's leave this and that of first_name as 200 like before to be consistent with that of Author.